src/@newrelic/gatsby-theme-newrelic/components/GlobalFooter.js

```diff
@@ -91,27 +83,26 @@ const GlobalFooter = ({
           />
         </Link>
         <div>
-          {repository && (
-            <CreateIssueButton
-              pageTitle={pageTitle}
-              variant={Button.VARIANT.OUTLINE}
-              size={Button.SIZE.SMALL}
-              labels={issueLabels}
-              instrumentation={{ component: 'GlobalFooter' }}
+          <Button
+            as={ExternalLink}
+            variant={Button.VARIANT.OUTLINE}
+            to="https://developer.newrelic.com/contribute-to-quickstarts/"
```

Comment:
According to [the docs](https://github.com/newrelic/gatsby-theme-newrelic/tree/main/packages/gatsby-theme-newrelic#externallink), I think we want to use `href` not `to`. Let's see if that solves the problem 🤷 
```suggestion
            href="https://developer.newrelic.com/contribute-to-quickstarts/"
```